tests/terraform_setup/ec2/terraform.tf

```diff
@@ -64,6 +61,7 @@ resource "aws_cloudformation_stack" "ecs_service" {
   depends_on = [
     aws_cloudformation_stack.vpc,
     aws_ecr_repository.demo-app-repository,
+    # aws_iam_role.parameter_store_read_role
```

Comment:
Why was this removed?

Reply:
The role was moved to the Cloudformation template